windows/ansi_reader.go

```diff
@@ -30,8 +30,8 @@ type ansiReader struct {
 
 // NewAnsiReader returns an io.ReadCloser that provides VT100 terminal emulation on top of a
 // Windows console input handle.
-func NewAnsiReader(nFile int) io.ReadCloser {
-	file, fd := winterm.GetStdFile(nFile)
+func NewAnsiReader(nFile int64) io.ReadCloser {
```

Comment:
This looks like a breaking change (if we want to follow SemVer for go modules), correct?

Reply:
Yes, unfortunately this is breaking.

Reply:
I think a review of the underlying win32 APIs to be sure of the type they take is required. e.g. if they take a HANDLE, it should explicitly be an intptr to be the correct size both for x86 and x64 architectures

Reply:
Could you check that, @saschagrunert ?

Reply:
I'm not sure if I get the point here. The function `GetStdFile` calls `syscall.GetStdHandle()` which requires an `int`:
https://github.com/Azure/go-ansiterm/blob/d6e3b3328b783f23731bc4d058875b0371ff8109/winterm/ansi.go#L176